spec/commands/remove_spec.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+RSpec.describe "bundle remove" do
+  context "remove a single gem from gemfile" do
+    it "when gem is present in gemfile" do
+      gemfile <<-G
+        source "file://#{gem_repo1}"
+
+        gem "rack"
+      G
+
+      bundle "remove rack"
```

Comment:
when the commands are expected to succeed, please use `bundle!`

Reply:
Sure. Just discovered the `bang` method.